### DISCUSSION TITLE
Please don't go with the 2d Mac/Ubuntu style doc.

### DISCUSSION THREAD

**Discussion by DaWindersGuy** [elementary/triage#589] [Ideas]
The dock is perfect the way it looks now. There's no reason to change it simply for following trends. Stand out among everything else like you used to.

**Comment by 1280px** [comment]
The dock that will ship in eOS 8 is brand new, re-written from scratch, it's not just a Plank with a new custom GTK3 theme... It seems there was an idea to change the appearance closer to curernt Plank appearance at some point, and there is even a PR that unblocks it, but it is rather stale: https://github.com/elementary/dock/pull/180

But agree, it feels latest eOS releases' UI is dictated by macOS' design trends more heavily — I wonder whether it is intentional move from UX team or just a sad coincidence.

**Reply by DaWindersGuy** [reply]
It's definitely intentional. They'd been working on it for a long time. I don't follow trends so this is all new to me.

**Reply by teamcons** [reply]
mostly a decision following a user survey of a few years ago.

it is brand new. It is themable. Im playing with the CSS. There just isnt a theme switcher now. Leave it some time.

**Reply by pckotzer** [reply]
Would be cool if i can select where i want the dock to be, like in 7. Iam more used to the left side. If you have something to select on the bottom it interferes and i accidentally select something on the dock. Also on wayland, if u use a full screen application via wine, and do volume up, it goes to the main screen and then the volume bar pops up, rather then popping up in the application itself.

**Reply by teamcons** [reply]
the issue with fullscreen is being addressed

**Reply by pckotzer** [reply]
Just tested, it still happens, it goes to main screen, instead of popping up in the application itself.

**Reply by teamcons** [reply]
> Just tested, it still happens, it goes to main screen, instead of popping up in the application itself.

Issues for notifications are reported https://github.com/elementary/notifications/issues
This topic is about the dock.

**Reply by 1280px** [reply]
> Iam more used to the left side

You still can't put it on sides? :O I've seen there was a protocol schema with different positioning options, so I just thought it was implemented months ago.

**Comment by danirabbit** [comment]
I guess I’m gonna close this as “outdated”, sorry! Hopefully when we roll out the workspace switcher it’ll make more sense as a design choice for you 🩷

**Reply by DaWindersGuy** [reply]
I've since stopped using eOS and stopped following the development of this distro as it's no longer trying to set itself apart but follow recent design trends. Thank you for your closing of this thread.
